tests/test_toa_writer.py

```diff
@@ -129,7 +134,7 @@ def test_write_pn_nan():
     t.write_TOA_file(f)
     f.seek(0)
     contents = f.read()
-    assert "nan" not in contents
+    assert "nan" not in contents.split()
```

Comment:
This error got triggered because my name is Abhimanyu Susobha*nan*. 😆

Reply:
We should make this even more robust because "Nan" is a Chinese name. This error will get triggered if a person named "Nan" runs this test, even with the ".split()" change.

Reply:
Fixed in 984b9f9a991ad8baf0dffcea5cfc301ef1db53ac